Hestia/Hestia.UnitTests/Hestia.UnitTests.csproj

```diff
@@ -0,0 +1,100 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
That is strange, you should be able to run them with the community edition. I have the community edition and I can run them.

Reply:
You can open the Test Explorer and run/see the tests with `Test -> Windows -> Test Explorer`. In case this is not working here are two screenshots of all the tests: 
![tests1](https://user-images.githubusercontent.com/22012675/39648445-85bcc1da-4fe2-11e8-8e7f-3ce1db4bad45.png)
![test2](https://user-images.githubusercontent.com/22012675/39648446-85d81822-4fe2-11e8-87f0-52aa1b970782.png)

Reply:
Hmm I think it's a visual studio mac thing. Thanks for the screenshot, why is the one test failing?

Reply:
That test is failing because it assumes that the Equals function can compare individual variables and not just the memory address of an object. This is currently not yet the case hence, the failure.